templates/users/new.rs.html

```diff
@@ -18,6 +18,7 @@ <h1>@i18n!(ctx.1, "Create an account")</h1>
             <input type="submit" value="@i18n!(ctx.1, "Create your account")" />
         </form>
     } else {
-<p class="center">@i18n!(ctx.1, "Apologies, but registrations are closed on this instance. Visit joinplu.me/#instances, to find a different one")</p>
+<p class="center">@i18n!(ctx.1, "Apologies, but registrations are closed on this instance.
+    <a href="https://joinplu.me/#instances">@i18n!(ctx.1, "Find a different one")</a>")</p>
```

Comment:
The link should be outside of the `i18n!` call (after the <p>, ideally wrapped in another one to make it centered too), otherwise we will probably end with something really weird (I didn't tested, but it will at least be hard for translators to read, and could result in rendering issue).

Reply:
Ahh, I think I understand now, sorry 👍 
Just pushed another commit to fix this, hopefully a final one for this pull request.